db/schema.rb

```diff
@@ -510,13 +903,447 @@
   end
 
+  create_table "pd_bookings", force: :cascade do |t|
+    t.string "jailnet_inmate_id"
+    t.string "initial_docket_id"
+    t.string "inmate_name"
+    t.string "inmate_aka"
+    t.datetime "birth_date", precision: nil
+    t.string "city_of_birth"
+    t.string "state_of_birth"
+    t.integer "current_age"
+    t.string "race"
+    t.string "gender"
+    t.integer "height"
+    t.float "weight"
+    t.string "hair_color"
+    t.string "eye_color"
+    t.string "build"
+    t.string "complexion"
+    t.string "facial_hair"
+    t.string "martial_status"
+    t.string "emergency_contact"
+    t.string "emergency_phone"
+    t.string "drivers_state"
+    t.string "drivers_license"
+    t.string "address1"
+    t.string "address2"
+    t.string "city"
+    t.string "state"
+    t.string "zip_code"
+    t.string "home_phone"
+    t.string "fbi_nbr"
+    t.string "osbi_nbr"
+    t.string "tpd_nbr"
+    t.integer "age_at_booking"
+    t.integer "age_at_release"
+    t.string "arrest_date"
+    t.string "arrest_by"
+    t.string "agency"
+    t.string "booking_date"
+    t.string "booking_by"
+    t.string "otn_nbr"
+    t.string "estimated_release_date"
+    t.string "release_date"
+    t.string "release_by"
+    t.string "release_reason"
+    t.string "weekend_server"
+    t.string "custody_level"
+    t.string "assigned_cell_id"
+    t.string "current_location"
+    t.string "booking_notes"
+    t.string "booking_alerts"
+    t.string "booking_trustees"
+  end
+
+  create_table "pd_offense_minutes", force: :cascade do |t|
+    t.bigint "offense_id", null: false
+    t.datetime "minute_date", precision: nil
+    t.string "minute"
+    t.string "minute_by"
+    t.string "judge"
+    t.string "next_proceeding"
+    t.index ["offense_id"], name: "index_pd_offense_minutes_on_offense_id"
+  end
+
+  create_table "pd_offenses", force: :cascade do |t|
+    t.bigint "booking_id", null: false
+    t.string "docket_id"
+    t.integer "offense_seq"
+    t.string "case_number"
+    t.string "offense_code"
+    t.string "offense_special_code"
+    t.string "offense_description"
+    t.string "offense_category"
+    t.string "court"
+    t.string "judge"
+    t.datetime "court_date", precision: nil
+    t.float "bond_amount"
+    t.string "bond_type"
+    t.integer "jail_term"
+    t.string "jail_sentence_term_type"
+    t.datetime "jail_conviction_date", precision: nil
+    t.datetime "jail_start_date", precision: nil
+    t.string "form41_filed"
+    t.string "docsentence_term"
+    t.string "docsentence_term_type"
+    t.datetime "docsentence_date", precision: nil
+    t.string "docnotified"
+    t.string "sentence_agent"
+    t.string "narative"
+    t.string "disposition"
+    t.datetime "disposition_date", precision: nil
+    t.datetime "entered_date", precision: nil
+    t.string "entered_by"
+    t.datetime "modified_date", precision: nil
+    t.string "modified_by"
+    t.index ["booking_id"], name: "index_pd_offenses_on_booking_id"
+  end
+
```

Comment:
This is a big schema change that is not related to this PR. Let's drop this so that our schema file stays clean 🧹